src/main/java/org/asamk/Signal.java

```diff
@@ -551,6 +555,50 @@ interface Group extends DBusInterface, Properties {
         void enableLink(boolean requiresApproval) throws Error.Failure;
     }
 
+    class StructIdentity extends Struct {
+
+        @Position(0)
+        DBusPath objectPath;
+
+        @Position(1)
+        String id;
```

Comment:
These should also be named `number` and `uuid` instead of `id`/`name`

Reply:
Ok. Question to the interface: Should we use the uuid as Dbus Path, or the human readable number?
Personally, I would prefer the number, with fallback to the uuid when the number is not known.

Reply:
I think it's fine this way.